### DISCUSSION TITLE
calendar unique value

### DISCUSSION THREAD

**Discussion by moemago03** [YetiForceCompany/YetiForceCRM#16030] [How to]
Hi,
How can I create a system that reserves the used calender slots? (This can be used for room booking, product rent, see user free slot, ecc).

Example: Mario has an appointment in his calendar for today at 16pm. When I try to assign a new appointment on the Mario calendar for today at 16pm it has to say me an error that tells "impossible to create the appointment because Mario has already another appointment at that day and time"

Is there a way to do this? maybe a module?

Thank a lot to who reply.

**Comment by vovpff** [comment]
I think what self made handler `editViewPreSave` can be used for your task. You can do it by yourself or charge to devs crew.
